float32 transBearing
float32 pitch
float32 TargetRange
float32 meanIntinsity
string  rawData
